server/public/pluginapi/store_test.go

```diff
@@ -54,13 +54,15 @@ func TestStore(t *testing.T) {
 		store := pluginapi.NewClient(api, driver).Store
```

Comment:
Can we have 2 separate sub-tests with different instances of store? One should call `GetMasterDB` first, the other should call `GetReplicaDB` first. The current changes trade off one code path for another.

Reply:
Done! added "master db fallback without get master first" test